core/server/common/src/main/java/alluxio/ExecutorServiceBuilder.java

```diff
@@ -9,11 +9,12 @@
  * See the NOTICE file distributed with this work for information regarding copyright ownership.
  */
 
-package alluxio.master;
+package alluxio;
```

Comment:
not a good practice to put classes in top level package
how about `alluxio.util`  ?

Reply:
@ZhuTopher  seems this is not addressed in the merged version?